sklearn/tests/test_multioutput.py

```diff
@@ -275,6 +275,10 @@ def test_multi_output_classification():
         assert_array_equal(list(forest_.predict_proba(X)),
                            list(predict_proba[i]))
 
+    # test classes_ attribute
+    [assert_array_equal(classes_true, classes_est) for
+     (classes_true, classes_est) in zip(classes, multi_target_forest.classes_)]
```

Comment:
This will pass if classes_ is an empty iterable. Check its type and length first

Reply:
Ideally we would check consistency of classes_ with other multi output estimators

Reply:
Thanks for the feedback @jnothman.

Based on your comments, I decided to write a new function to test both MultiOutputClassifier and ClassifierChain and also compare them to natively-supported multioutput classification.